inspire_mitmproxy/interaction.py

```diff
@@ -71,6 +75,20 @@ def from_file(cls, interaction_file: Optional[Path]) -> 'Interaction':
             callbacks=interaction_dict.get('callbacks'),
         )
 
+    def to_dict(self) -> dict:
+        serialized_interaction: Dict[str, Any] = {
+            'request': self.request.to_dict(),
+            'response': self.response.to_dict(),
+        }
+
+        if self.match:
+            serialized_interaction['match'] = self.match
+
+        if self.callbacks:
+            serialized_interaction['callbacks'] = self.callbacks
```

Comment:
Add these two sections even if empty, to hint anyone that tries to manually edit the files after.